vexim/userchangesubmit.php

```diff
@@ -28,7 +28,7 @@
     }
   }
 
-  if ($_POST['realname'] !== "") {
+  if (isset($_POST['realname'])) {
```

Comment:
IMO, checking for non-empty realname was a good thing to do. Perhaps you should use both `isset()` and `!= ''`?